internal/oci/file.go

```diff
@@ -433,6 +427,65 @@ func (s *Store) buildLayers(ctx context.Context, store oras.Target, src fs.FS) (
 	return layers, nil
 }
 
+func (s *Store) Copy(ctx context.Context, src, dst Reference) (Bundle, error) {
+	if src.Reference.Reference == "" {
+		return Bundle{}, fmt.Errorf("source bundle: %w", ErrReferenceRequired)
+	}
+
+	if dst.Reference.Reference == "" {
+		return Bundle{}, fmt.Errorf("destination bundle: %w", ErrReferenceRequired)
+	}
+
+	srcTarget, err := s.getTarget(src)
+	if err != nil {
+		return Bundle{}, err
+	}
+
+	dstTarget, err := s.getTarget(dst)
+	if err != nil {
+		return Bundle{}, err
+	}
+
+	desc, err := oras.Copy(
+		ctx,
+		srcTarget,
+		src.Reference.Reference,
+		dstTarget,
+		dst.Reference.Reference,
+		oras.DefaultCopyOptions)
+	if err != nil {
+		return Bundle{}, err
+	}
+
+	rd, err := dstTarget.Fetch(ctx, desc)
+	if err != nil {
+		return Bundle{}, err
+	}
+
+	data, err := io.ReadAll(rd)
+	if err != nil {
+		return Bundle{}, err
+	}
+
+	var man v1.Manifest
+	if err := json.Unmarshal(data, &man); err != nil {
+		return Bundle{}, err
+	}
```

Comment:
Take it or leave it.

```go
json.NewDecoder(rd).Decode(&man)
```

Reply:
Ill get it in the next one since this is not going into main 👍 Good shout.